### DISCUSSION TITLE
Allow groups to be half width for more complex layouts

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13378] [Feature Requests]
This would be great for slightly more advanced form layouts!

**Comment by benhaynes** [comment]
Could you explain which component you mean by "panels"?

**Reply by sxsws** [reply]
I just checked, directus calls them "Groups"! So, the accordian, Detail Group, Raw Group.

**Reply by benhaynes** [reply]
Ahh, that makes sense. Could be interesting!

**Comment by sxsws** [comment]
see https://github.com/directus/directus/discussions/13378 - half-width groups are now seemingly possible with manual hacks

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
